src/tests/ftest/pool/create_all_vm.py

```diff
@@ -63,25 +63,25 @@ def test_one_pool(self):
         :avocado: tags=pool,pool_create_all
         :avocado: tags=pool_create_all_one_vm,test_one_pool
         """
-        self.log.info("Test  basic pool creation with full storage")
+        self.log.info("Test basic pool creation with full storage")
 
         create_time = self.create_one_pool()
-        self.log.debug("Created one pool with 100%% of the available storage "
-                "in %f seconds", create_time)
+        self.log.debug("Created one pool with 100%% of the available storage in %f seconds",
+                       create_time)
```

Comment:
```suggestion
        self.log.debug(
            "Created one pool with 100%% of the available storage in %f seconds",
            create_time)
```
For Python, we usually break line at the open parenthesis.

Reply:
- [ ] fix invalid coding style